ibis/tests/all/test_window.py

```diff
@@ -45,7 +45,7 @@
             lambda t, win: t.float_col.ntile(buckets=7).over(win),
             lambda t: t,
             id='ntile',
-            marks=pytest.mark.xfail,
+            marks=pytest.mark.xfail
```

Comment:
The comma here was fine as is. Was this removed by `black`?

Reply:
I don't remember ... I made some changes that to play with tests ... so maybe when I rebased and fix ...  I lost the comma ... I will put that again :)